src/app/components/msel-list/msel-list.component.html

```diff
@@ -133,7 +49,49 @@
       </button>
     </div>
   </div>
-
+<ng-container>

```

Comment:
I think this ng-container may be unnecessary here.

Reply:
You are right, it is unnecessary.